radixdlt-java/src/main/java/com/radixdlt/client/core/atoms/Atom.java

```diff
@@ -202,4 +216,8 @@ public int hashCode() {
 	public String toString() {
 		return "Atom (" + this.getHid().toString() + ")";
 	}
+
+	public Map<String, String> getMetaData() {
+		return this.metaData;
```

Comment:
I've noticed that we sometimes prefer to return unmodifiable copies of `List`'s and `Set`'s. I'm not sure if that pattern would also apply to `Map`'s as well, or if it's relevant in this case at all (do we expose this to user facing abstractions, for example?)

What's your take @mpsandiford?

Reply:
I think it would make a lot of sense here, what's the best way to do that?

Reply:
`ImmutableMap` for the declaration of `this.metaData` would be best.